network-api/networkapi/mozfest/templates/partials/primary_heroguts.html

```diff
@@ -0,0 +1,30 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+
+<div id="hero">
+  <div class="banner">
+    {% if page.banner %}
```

Comment:
Just to keep partials legible, it might be worth adding this separation to the foundation `primary_heroguts.html` as well.

Reply:
Agreed.

Reply:
https://github.com/mozilla/foundation.mozilla.org/issues/3166